packages/toolpad-app/src/components/AppEditor/BindingEditor.tsx

```diff
@@ -155,11 +160,11 @@ export function BindingEditor<V>({
 
   const TooltipComponent = error ? ErrorTooltip : Tooltip;
   const tooltipTitle: string =
-    error ?? (hasBinding ? `Update "${label}" binding` : `Bind "${label}"`);
+    error ?? (hasBinding ? `Update "${label}" binding...` : `Bind "${label}"...`);
```

Comment:
Thanks for the fix. If you're onboard with using an ellipsis in tooltips to indicate that a button opens a dialog, then perhaps the unicode "…" is preferable to  three full-stops/periods "..."?

Reply:
https://github.com/mui/mui-toolpad/pull/519#event-6727268978